### ISSUE TITLE
Teilnehmer Spielergebnisse in DB eintragen

### ISSUE THREAD

**Issue by DiFried** [Mosherfist/Boule-Petanque#5]
Spielergebnisse eintragen
> „_gewonnene_Spiele_“ Für jedes gewonnene Spiel. Zähler um 1 hochsetzen
> „_Punkte_“ nach jedem Spiel die Diff-Punkte aufsummieren
>> Beispiel: 13:5 = Diff.Punkte +8
>> Beispiel: 5:13 = Diff. Punkte -8

**Comment by Mosherfist** [comment]
@DiFried Wie ich gesehen habe ist im Turnier Menü nicht mehr der Punkt "Spielergebnisse eintragen" enthalten, soll ich mich darum kümmern?

**Comment by DiFried** [comment]
Hallo Uwe,

Ich verstehe Deine Anfrage nicht.- Im Code unter „def Ergebnisse:“ letzter Abschnitt sind alle notwendigen Eintragungen.
Lass uns später nochmals darüber sprechen.

Mit freundlichen Grüßen
Dieter Friedrich

**Comment by Mosherfist** [comment]
@DiFried Ja, das habe ich gesehen, aber in der Methode "def turnier_menue" fehlt der Eintrag im Menü, soll der Code wieder ergänzt werden?

**Comment by DiFried** [comment]
Uwe,

Ich glaube, ich weiß was Du meinst.
Ich habe es korrigiert, und alles sollte nun richtig laufen.
Schönen Abend noch, bis Sonntag..

Mit freundlichen Grüßen
Dieter Friedrich
